astroquery/ipac/irsa/core.py

```diff
@@ -60,10 +76,52 @@ def query_tap(self, query, *, maxrec=None):
         log.debug(f'TAP query: {query}')
         return self.tap.search(query, language='ADQL', maxrec=maxrec)
 
+    def query_sia(self, *, pos=None, band=None, time=None, pol=None,
+                  field_of_view=None, spatial_resolution=None,
+                  spectral_resolving_power=None, exptime=None,
+                  timeres=None, publisher_did=None,
+                  facility=None, collection=None,
+                  instrument=None, data_type=None,
+                  calib_level=None, target_name=None,
+                  res_format=None, maxrec=None,
+                  **kwargs):
+        """
+        Use standard SIA2 attributes to query the IRSA SIA service.
+
+        Parameters
+        ----------
+        _SIA2_PARAMETERS
```

Comment:
Does this get automatically expanded somehow?

Reply:
good point, I'll do the same what we do for `alma`